app/queries/screening_list/query.rb

```diff
@@ -16,6 +16,7 @@ def initialize(options = {})
       @start_date = options[:start_date] if options[:start_date].present?
       @issue_date = options[:issue_date] if options[:issue_date].present?
       @expiration_date = options[:expiration_date] if options[:expiration_date].present?
+      @phonetics = options[:phonetics] if options[:phonetics].present?
```

Comment:
I feel you should do `@phonetics = true if options[:phonetics].present?`, so that later you have conditions like:
```
if @phonetics
  # phonetics stuff
end
```
rather than:
```
if @phonetics == '1'
  # phonetics stuff
end
```